From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Leandro Dorileo <leandro.maciel.dorileo@intel.com>
Date: Thu, 19 Sep 2019 14:44:36 -0700
Subject: [PATCH] preserve timestamps

Port note: the pack.c bits were added upstream by commit:
b8a54d6a1e9bb6140b6b47e23dc707e4b967537e

Ported-by: Thiago Macieira <thiago.macieira@intel.com>
Signed-off-by: Leandro Dorileo <leandro.maciel.dorileo@intel.com>
---
 scripts/brp-strip                | 2 +-
 scripts/brp-strip-comment-note   | 2 +-
 scripts/brp-strip-shared         | 2 +-
 scripts/brp-strip-static-archive | 2 +-
 scripts/find-debuginfo.sh        | 2 +-
 5 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/scripts/brp-strip b/scripts/brp-strip
index 5e64566..b56957f 100755
--- a/scripts/brp-strip
+++ b/scripts/brp-strip
@@ -16,5 +16,5 @@ for f in `find "$RPM_BUILD_ROOT" -type f \( -perm -0100 -or -perm -0010 -or -per
         grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug"  | \
 	grep -v ' shared object,' | \
 	sed -n -e 's/^\(.*\):[ 	]*ELF.*, not stripped.*/\1/p'`; do
-	$STRIP -g "$f" || :
+	$STRIP -p -g "$f" || :
 done
diff --git a/scripts/brp-strip-comment-note b/scripts/brp-strip-comment-note
index 833ac78..f668ba3 100755
--- a/scripts/brp-strip-comment-note
+++ b/scripts/brp-strip-comment-note
@@ -22,5 +22,5 @@ for f in `find "$RPM_BUILD_ROOT" -type f \( -perm -0100 -or -perm -0010 -or -per
 		grep ALLOC >/dev/null; then
 		note=
 	fi
-	$STRIP -R .comment $note "$f" || :
+	$STRIP -p -R .comment $note "$f" || :
 done
diff --git a/scripts/brp-strip-shared b/scripts/brp-strip-shared
index 51d10d5..b9eaa5f 100644
--- a/scripts/brp-strip-shared
+++ b/scripts/brp-strip-shared
@@ -21,5 +21,5 @@ for f in `find "$RPM_BUILD_ROOT" -type f -a -exec file {} \; | \
         grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug"  | \
 	grep ' shared object,' | \
 	sed -n -e 's/^\(.*\):[ 	]*ELF.*, not stripped.*/\1/p'`; do
-	$STRIP --strip-unneeded "$f"
+	$STRIP -p --strip-unneeded "$f"
 done
diff --git a/scripts/brp-strip-static-archive b/scripts/brp-strip-static-archive
index ddd3b24..3f3542c 100755
--- a/scripts/brp-strip-static-archive
+++ b/scripts/brp-strip-static-archive
@@ -16,5 +16,5 @@ for f in `find "$RPM_BUILD_ROOT" -type f -a -exec file {} \; | \
         grep -v "^${RPM_BUILD_ROOT}/\?usr/lib/debug"  | \
 	grep 'current ar archive' | \
 	sed -n -e 's/^\(.*\):[ 	]*current ar archive/\1/p'`; do
-	$STRIP -g "$f"
+	$STRIP -p -g "$f"
 done
diff --git a/scripts/find-debuginfo.sh b/scripts/find-debuginfo.sh
index ac24beb..c3615e7 100755
--- a/scripts/find-debuginfo.sh
+++ b/scripts/find-debuginfo.sh
@@ -237,7 +237,7 @@ strip_to_debug()
   application/x-executable*) g=-g ;;
   application/x-pie-executable*) g=-g ;;
   esac
-  eu-strip --remove-comment $r $g ${keep_remove_args} -f "$1" "$2" || exit
+  eu-strip -p --remove-comment $r $g ${keep_remove_args} -f "$1" "$2" || exit
   chmod 444 "$1" || exit
 }
 
